### DISCUSSION TITLE
780m support

### DISCUSSION THREAD

**Discussion by Adriankhl** [YellowRoseCx/koboldcpp-rocm#40] [Q&A] [answered]
Hi, I am running a laptop with 780m apu. I am curious about running the rocm version, but the gfx1103 support is not included in the compilation process so I cannot run `hiprocm`. I did attempt to build it myself but I failed because of https://github.com/ROCm/ROCm/issues/2785. @YellowRoseCx How do you get around with the issue?

**Comment by YellowRoseCx** [comment, marked as answer]
> Hi, I am running a laptop with 780m apu. I am curious about running the rocm version, but the gfx1103 support is not included in the compilation process so I cannot run `hiprocm`. I did attempt to build it myself but I failed because of https://github.com/ROCm/ROCm/issues/2785. @YellowRoseCx How do you get around with the issue?

Are you on Windows or Linux? Because on Windows, I'm getting that same exact error which is why I haven't been able to build the latest version for Windows. But with Linux using the makefile, it works just fine doing ``make LLAMA_HIPBLAS=1 -j4``

If you're on Linux, try using the command ``HSA_OVERRIDE_GFX_VERSION=11.0.0`` before running koboldcpp.py after compiling, But if you're on Windows, then I believe the only way it could possibly work as if you were to compile the TensileLibraries yourself, but even then, it might not be supported.

**Reply by Adriankhl** [reply]
Thanks for the information. This is a windows machine because I also want to experiment with directml. So I probably need to wait for a better support from AMD (which may never come, I will keep checking https://github.com/ROCm/ROCm/discussions/2631). In the meanwhile I think the vulkan backend is very interesting.
